### PR TITLE
Review content & embed tutorial videos

### PR DESCRIPTION
**Pull request by tomwarwick**
This commit contains changes to the menu pages and has the new video
tutorials embedded.

### CODE REVIEW

File: Documentation/AccessControl/Index.rst [hunk 1]
```diff
@@ -16,11 +16,38 @@ period, all the way through to requiring visitors to log in to view content.
...
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: :ref:`Frontend Login<frontend-login>`
+
+         .. container:: card-body
+
+            You can specify a period of time for content to be visible.
```
Comment (brotkrueml) [inline Documentation/AccessControl/Index.rst:44]: The text does not match the header?

File: Documentation/Concepts/Index.rst [hunk 2]
```diff
@@ -8,10 +8,71 @@ Concepts
...
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: :ref:`Accessibility<accessibility>`
+
+         .. container:: card-body
+
+            In an introduction to writing accessible content with links to third party resources.
```
Comment (brotkrueml) [inline Documentation/Concepts/Index.rst:47]:
```suggestion
            An introduction to writing accessible content with links to third party resources.
```

File: Documentation/ContentElements/Index.rst [hunk 3]
```diff
@@ -7,15 +7,105 @@
...
 
+.. container:: row m-0 p-0
+
+   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: :ref:`Creating Content<content-creating>`
```
Comment (brotkrueml) [inline Documentation/ContentElements/Index.rst:18]: We use sentence case for headers.

File: Documentation/HelpInside/Index.rst [hunk 4]
```diff
@@ -44,6 +44,21 @@ Help section, then select the :guilabel:`TYPO3 Online Documentation` module.
...
+Tooltips
+========
+
+Hover your mouse pointer over buttons and icons to display a short description of the function.
+
+.. figure:: /Images/ManualScreenshots/Help/Tooltip.png
+   :alt: Tooltip for the Forms module
+   :class: with-border
+
+   See the Tooltip help for the Forms module
```
Comment (brotkrueml) [inline Documentation/HelpInside/Index.rst:60]: Why "Tooltip" with upper T?

File: Documentation/Index.rst [hunk 5]
```diff
@@ -1,14 +1,12 @@
...
+===================
 
-Once you have installed TYPO3 using our :doc:`Getting Started Guide <t3start:Index>`;
-the next logical step is to log in to the backend of the CMS and start adding pages and create
-some content.
-
-In the TYPO3 world we often call users who carry out these tasks "Editors".
+Once you have installed TYPO3 using the :doc:`Getting Started Guide <t3start:Index>`;
+the next step is to log in to the backend of the CMS and start adding pages and create
+some content. In the TYPO3 world we often call users who carry out these tasks "Editors".
```
Comment (brotkrueml) [inline Documentation/Index.rst:9]: Why upper E in "Editors"?

File: Documentation/ListModule/Index.rst [hunk 6]
```diff
@@ -19,11 +19,63 @@ For example, a typical page would consist of several content elements that
...
 
+.. container:: row m-0 p-0
+
+   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: :ref:`Using The List Module<using-the-list-module-effectively>`
```
Comment (brotkrueml) [inline Documentation/ListModule/Index.rst:30]: Also here: sentence case in headers

File: Documentation/ListModule/Index.rst [hunk 7]
```diff
@@ -19,11 +19,63 @@ For example, a typical page would consist of several content elements that
...
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: :ref:`The Clipboard<clipboard>`
+
+         .. container:: card-body
+
+            Every time a record is copied in the TYPO3, it appears in the clipboard found at the bottom of the list module screen.
```
Comment (brotkrueml) [inline Documentation/ListModule/Index.rst:46]:
```suggestion
            Every time a record is copied in TYPO3, it appears in the clipboard found at the bottom of the list module screen.
```

File: Documentation/ListModule/Index.rst [hunk 8]
```diff
@@ -19,11 +19,63 @@ For example, a typical page would consist of several content elements that
...
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: :ref:`Advanced Clipboard Usage<advanced_clipboard_usage>`
+
+         .. container:: card-body
+
+            Find out how to make get the most out of the clipboard with an overview of some
```
Comment (brotkrueml) [inline Documentation/ListModule/Index.rst:59]:
```suggestion
            Find out how to get the most out of the clipboard with an overview of some
```

File: Documentation/ListModule/Index.rst [hunk 9]
```diff
@@ -19,11 +19,63 @@ For example, a typical page would consist of several content elements that
...
+            its extra features.
```
Comment (brotkrueml) [inline Documentation/ListModule/Index.rst:60]:
```suggestion
            of its extra features.
```

File: Documentation/ListModule/Index.rst [hunk 10]
```diff
@@ -19,11 +19,63 @@ For example, a typical page would consist of several content elements that
...
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: :ref:`Mass Editing<mass-editing>`
+
+         .. container:: card-body
+
+            The List module makes it possible to display the content of several fields at once and gives you the ability edit several records with one action.
```
Comment (brotkrueml) [inline Documentation/ListModule/Index.rst:73]:
```suggestion
            The List module makes it possible to display the content of several fields at once and gives you the ability to edit several records with one action.
```

File: Documentation/Login/Index.rst [hunk 11]
```diff
@@ -2,22 +2,18 @@
...
-   Make sure JavaScript and cookies are enabled in your browser as they are
-   required by the TYPO3 Backend.
-
-Enter your domain name into the address bar of your internet browser and
-append "/typo3" to it to get to the "log in" page of your site.
+Enter your domain name into the address bar of your browser and
+append "/typo3" to the end of it to access the backend login page.
 For example: :samp:`http://www.example.org/typo3`.
 
+Check that JavaScript and cookies are enabled in your browser as they are required by TYPO3's Backend.
```
Comment (brotkrueml) [inline Documentation/Login/Index.rst:16]:
```suggestion
Check that JavaScript and cookies are enabled in your browser as they are required by TYPO3's backend.
```

File: Documentation/Pages/AddingMultiple/Index.rst [hunk 12]
```diff
@@ -3,46 +3,41 @@
 
 .. _pages-multiple:
 
-=============================
-Adding multiple pages at once
-=============================
+=======================
+Creating Multiple Pages
```
Comment (brotkrueml) [inline Documentation/Pages/AddingMultiple/Index.rst:7]:
```suggestion
Creating multiple pages
```

File: Documentation/Pages/CreatingPages/Index.rst [hunk 13]
```diff
@@ -4,39 +4,49 @@
 .. _pages-creating:
 
 ==============
-Create pages
+Creating Pages
```
Comment (brotkrueml) [inline Documentation/Pages/CreatingPages/Index.rst:7]:
```suggestion
Creating pages
```

File: Documentation/Pages/CreatingPages/Index.rst [hunk 14]
```diff
@@ -4,39 +4,49 @@
...
+
+------------
+
+There are two ways to add a new page to your site. It can be done either by dragging the new page
+icon to the desired location in the page tree or by right-clicking on
+an existing page and using the contextual menu.
+
+To start adding pages:
+
+* Select the :guilabel:`Web` module in the backend.
```
Comment (brotkrueml) [inline Documentation/Pages/CreatingPages/Index.rst:20]:
```suggestion
* Select the :guilabel:`Web > Page` module in the backend.
```

File: Documentation/Pages/Index.rst [hunk 15]
```diff
@@ -7,11 +7,82 @@
...
 
+.. container:: row m-0 p-0
+
+   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: :ref:`Creating Pages<pages-creating>`
```
Comment (brotkrueml) [inline Documentation/Pages/Index.rst:18]: Sentence case please (also below).

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Documentation/AccessControl/Index.rst (hunk 1 above)
Comment (tomwarwick) [reply Documentation/AccessControl/Index.rst]: Correct - this should be resolved now. Thank you

**Comment by tomwarwick** [comment]
Manual back-port completed.

**Pull request merged**